diff -urN libxlsxwriter-RELEASE_0.8.7/CMakeLists.txt libxlsxwriter-RELEASE_0.8.7.patched/CMakeLists.txt
--- libxlsxwriter-RELEASE_0.8.7/CMakeLists.txt	2019-06-21 01:07:52.000000000 +0200
+++ libxlsxwriter-RELEASE_0.8.7.patched/CMakeLists.txt	2019-07-18 12:15:34.262806694 +0200
@@ -103,6 +103,7 @@
 option(USE_SYSTEM_MINIZIP "Use system minizip installation" OFF)
 option(USE_STANDARD_TMPFILE "Use the C standard library's tmpfile()" OFF)
 option(IOAPI_NO_64 "Disable 64-bit filesystem support" OFF)
+option(USE_DOUBLE_FUNCTION "use double function (avoid locale problem)" OFF)
 if(MSVC)
     option(USE_STATIC_MSVC_RUNTIME "Use the static runtime library" OFF)
 endif()
@@ -153,6 +154,10 @@
     endforeach()
 endif()
 
+IF(USE_DOUBLE_FUNCTION)
+    ADD_DEFINITIONS(-DUSE_DOUBLE_FUNCTION)
+ENDIF(USE_DOUBLE_FUNCTION)
+
 # INCLUDES
 # --------
