diff -Naur libxlsxwriter-RELEASE_1.0.6.orig/CMakeLists.txt libxlsxwriter-RELEASE_1.0.6/CMakeLists.txt
--- libxlsxwriter-RELEASE_1.0.6.orig/CMakeLists.txt	2021-05-28 10:05:13.000000000 -0500
+++ libxlsxwriter-RELEASE_1.0.6/CMakeLists.txt	2021-06-01 01:36:41.837625000 -0500
@@ -127,11 +127,16 @@
 option(USE_OPENSSL_MD5 "Build libxlsxwriter with the OpenSSL MD5 lib" OFF)
 option(USE_FMEMOPEN "Use fmemopen() in place of some temporary files" OFF)
 option(IOAPI_NO_64 "Disable 64-bit filesystem support" OFF)
+option(USE_DOUBLE_FUNCTION "use double function (avoid locale problem)" OFF)
 
 if(MSVC)
     option(USE_STATIC_MSVC_RUNTIME "Use the static runtime library" OFF)
 endif()
 
+IF(USE_DOUBLE_FUNCTION)
+    ADD_DEFINITIONS(-DUSE_DOUBLE_FUNCTION)
+ENDIF(USE_DOUBLE_FUNCTION)
+
 if(DEFINED ENV{${ZLIB_ROOT}})
     set(ZLIB_ROOT $ENV{ZLIB_ROOT})
 endif()
