diff -Naur libxlsxwriter-RELEASE_0.9.1.orig/CMakeLists.txt libxlsxwriter-RELEASE_0.9.1.patched/CMakeLists.txt
--- libxlsxwriter-RELEASE_0.9.1.orig/CMakeLists.txt	2019-12-27 05:40:12.000000000 -0600
+++ libxlsxwriter-RELEASE_0.9.1.patched/CMakeLists.txt	2020-01-02 13:49:03.275192359 -0600
@@ -119,6 +119,7 @@
 option(USE_STANDARD_TMPFILE "Use the C standard library's tmpfile()" OFF)
 option(USE_NO_MD5 "Build libxlsxwriter without third party MD5 lib" OFF)
 option(IOAPI_NO_64 "Disable 64-bit filesystem support" OFF)
+option(USE_DOUBLE_FUNCTION "use double function (avoid locale problem)" OFF)
 
 if(MSVC)
     option(USE_STATIC_MSVC_RUNTIME "Use the static runtime library" OFF)
@@ -173,6 +174,10 @@
     endforeach()
 endif()
 
+IF(USE_DOUBLE_FUNCTION)
+    ADD_DEFINITIONS(-DUSE_DOUBLE_FUNCTION)
+ENDIF(USE_DOUBLE_FUNCTION)
+
 # INCLUDES
 # --------
 enable_language(CXX)
